news/2018-05-16-twc-finals-recap.md

```diff
@@ -0,0 +1,52 @@
+
+**Note: Due to an unfortunate situation surrounding foul play, team United States has forfeited out of the TWC. We won't go over their matches in this recap. You can read about the situation [in this thread](https://osu.ppy.sh/community/forums/topics/743380).**
+
+### Finals Matches
+
+**In the first match, Canada claimed their top 4 placing by taking down South Korea 7-3.** The Koreans dominated the speedy nomod picks [Nanoka - Tsukimade Todoke, Fushi no Kemuri](https://osu.ppy.sh/beatmapsets/763774#taiko/1605963) and [Lily - Scarlet Rose](https://osu.ppy.sh/beatmapsets/195238#taiko/463330), but that only got them so far. Having reached the Finals as the only remaining mid-seed, this hailed the end of South Korea's spectacular run this TWC. Just imagine what they could accomplish with a bigger roster in future cups.
+
+History was simultaneously repeated and made in the Winners Finals match between Japan vs. Taiwan in one of the most exciting matches we've seen thus far. Everyone was on the edge of their seats as the increasing chinks in Japan's armor turned into an unprecedented 3 - 3 tie halfway into the match. Japan however regained their composure and pulled ahead to end with a 7 - 3 victory. With Japan's armor not as impenetrable as we thought, the stage was set for the Grand Finals to be more exciting than we've seen in a long time.
+
+Due to the forfeit of United States, Indonesia got another chance against Canada in the match for 3rd place. Unfortunately, one of the key players from Indonesia were missing which left the rest of Team Indonesia not on par with Canada's synergy. **The Canadians swept their way into top 3 with a 7-0 sweep.**
+
+When Canada went up against Taiwan, their memorization game was on point as they opened the match with a win on the SV map [cranky - R176](https://osu.ppy.sh/beatmaps/1623411). After that, Taiwan remembered their warmup map [Dave Rodgers - Deja Vu](https://osu.ppy.sh/beatmaps/1637925). **They suddenly remembered that in order to keep up with tradition, they'd have to make it to the Grand Finals.** In order to get there, they hit full throttle to the point where Canada could not keep up. Taiwan had secured their Grand Finals spot over Canada 7-1.
+
+As for the Grand Finals, Taiwan was still going at full speed since their last match, as they won the first 3 points against Japan. If that in itself isn't a historical achievement, Japan had to suffer 2 more maps lost as they clawed their way back to a 7-5 victory. Taiwan truly played their hearts out and made the Grand Final one of the most even matches of the tournament. Taiwan may have lost the match, but at least they ended the tournament with a win on the tiebreaker showcase: 
+
+![](/wiki/shared/news/2018-05-07-twc-finals-recap/japan-taiwan-tb.jpg)
+
+And that wraps up all the matches for this year! Here's how the entire elimination bracket went down:
+
+![](/wiki/shared/news/2018-05-07-twc-finals-recap/bracket.png)
```

Comment:
This bracket is missing P, AE, and AD.. unless it's supposed to be like that?
![image](https://user-images.githubusercontent.com/14264992/40147771-a9d908d8-5930-11e8-8600-eba7ed5919a4.png)